src/test/kotlin/no/nav/syfo/testutil/generator/KafkaOppfolgingstilfellePersonGenerator.kt

```diff
@@ -36,7 +37,7 @@ fun generateKafkaOppfolgingstilfellePerson(
             ),
         ),
         referanseTilfelleBitUuid = UUID.randomUUID().toString(),
-        referanseTilfelleBitInntruffet = nowUTC().minusDays(1),
+        referanseTilfelleBitInntruffet = nowUTC().minusDays(1).truncatedTo(ChronoUnit.MILLIS),
```

Comment:
Måtte trunkere til ms her for å gjenskape problemet i testkoden (siden det ser ut til at tidsstempelet blir noe trunkert i alle fall ved skriving til databasen og lesing tilbake).